prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/SC/SC-receiver-transform.yml

```diff
@@ -74,6 +74,10 @@ elements:
     value: [ '""' ]
     hl7Spec: [ /PATIENT_RESULT/PATIENT/PID-5-7 ]
 
+  - name: observation-result-with-aoe
```

Comment:
I don't think this is working as intended. I need to fix this for HI as well. If the goal is for AOEs to be sent as OBX segments this is not needed

Reply:
Thank you.  I just don't need to merge this PR, then.